java/Validator.java

```diff
@@ -0,0 +1,14 @@
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.XMLConstants;
+import java.io.File;
+import java.io.IOException;
+import org.xml.sax.SAXException;
+
+public class Validator {
+	public static void main(String[] args) throws IOException, SAXException {
+		var schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+		var validator = schemaFactory.newSchema(new StreamSource(new File("schemas/edmx.xsd"))).newValidator();
```

Comment:
Where is `edm.xsd` read from?

Reply:
Have removed that, `edm.xsd` is now read automatically from the `xs:import` statement.